discovery_engine_api_tests/model/properties.py

```diff
@@ -0,0 +1,6 @@
+from model.base.model_base import ModelBase
+
+
+class Properties(ModelBase):
+    def __init__(self, title):
+        self.title = title
```

Comment:
:question: / :pushpin:  couldn't we just use kwargs or similar to allow semi arbitrary choices of properties to test with.

Reply:
@rustonaut good point